src/Workspaces/CSharp/Portable/Recommendations/CSharpRecommendationServiceRunner.cs

```diff
@@ -494,7 +498,7 @@ private RecommendedSymbols GetSymbolsOffOfConditionalReceiver(ExpressionSyntax o
             // If we're showing instance members, don't include nested types
             var namedSymbols = excludeStatic
                 ? symbols.WhereAsArray(s => !(s.IsStatic || s is ITypeSymbol))
-                : symbols;
+                : (abstractStaticsOnly ? symbols.WhereAsArray(s => s.IsAbstract && s.IsStatic) : symbols);
```

Comment:
shouldn't the static check alreacy have happened because we passed excludeInstance to GetMemberSymbols.  so we only need to check abstracts here, right?